OpenRA.Mods.D2k/SpriteLoaders/R8Loader.cs

```diff
@@ -22,6 +22,8 @@ public class R8Loader : ISpriteLoader
 	{
 		class R8Frame : ISpriteFrame
 		{
+			public SpriteFrameType Type { get; set; }
+
```

Comment:
Minor consistency nit (not a blocker): This is the only loader you added an empty line to. Might want to drop this one while adressing @abcdefg30's points.